src/sentry/integrations/msteams/utils.py

```diff
@@ -39,7 +39,9 @@ def get_channel_id(organization, integration_id, name):
         member_list = members.get("members")
         continuation_token = members.get("continuationToken")
 
-        filtered_members = list(filter(lambda x: x.get("name") == name, member_list))
+        filtered_members = list(
```

Comment:
Lol, damn, does msteams not provide a proper search api either?

Reply:
@wedamija No, that would be too easy and smart